ncm-opennebula/src/main/pan/components/opennebula/schema.pan

```diff
@@ -310,19 +310,39 @@ type opennebula_datastore = {
 type opennebula_vnet = {
     "name" : string
     "bridge" : string
-    "gateway" : type_ipv4
-    "dns" : type_ipv4
-    "network_mask" : type_ipv4
+    "vn_mad" : string = 'dummy' with match (SELF, '^(802.1Q|ebtables fw|ovswitch|vxlan|vcenter|dummy)$')
+    "gateway" ? type_ipv4
+    "gateway6" ? type_network_name
+    "dns" ? type_ipv4
+    "network_mask" ? type_ipv4
+    "network_address" ? type_ipv4
+    "guest_mtu" ? long
+    "context_force_ipv4" ? boolean
+    "search_domain" ? string
     "bridge_ovs" ? string
     "vlan" ? boolean
     "vlan_id" ? long(0..4095)
     "ar" ? opennebula_ar
 };
 
+@documentation{
+Set OpenNebula regular users
+and their primary groups.
+By default new users are assigned to users group.
```

Comment:
@stdweird yes, OpenNebula always generates 2 groups after the service installation. `oneadmin` group to be used only by user admins and a generic `users` group. By default any new user is always assigned to `users` as primary group.